server/read-page.js

```diff
@@ -12,11 +12,15 @@ async function readPage (path) {
     return cache[f]
   }
 
-  const source = await fs.readFile(f, 'utf8')
-  const { component } = JSON.parse(source)
+  let content = await fs.readFile(f, 'utf8')
 
-  cache[f] = component
-  return component
+  if (f.slice(-5) === '.json') {
```

Comment:
Oh! wait. But for here, it's always `.json`. 
If there's no `.json` file, we need to throw an error.
Then I think we don't need a change here.